backend/src/main/validation/UserValidation.ts

```diff
@@ -0,0 +1,22 @@
+import Joi, { ObjectSchema } from 'joi';
+
+/**
+ * check the validity of an user, used when we make a POST request to /users
+ */
+export class UserValidation {
+    static schema: ObjectSchema = Joi.object({
+        username: Joi.string().required(),
+        firstName: Joi.string().required(),
+        lastName: Joi.string().required(),
+        email: Joi.string().required(),
```

Comment:
I saw that `Joi` has this additional module for email validation: https://joi.dev/module/address/#install.

Do you want to try this out?

I think there also might be `Joi.string().email()`.

Reply:
Ok, I'll try!